src/rules/unused_callbacks.erl

```diff
@@ -0,0 +1,64 @@
+%% @doc A rule to detect unused callbacks.
+%%      <p>This rule will check all callbacks defined in a module and find
+%%      those that are not used anywhere in the module itself.</p>
+%%      <p>It will emit a warning if it can't find the callback's atom name
+%%      being used anywhere within a module. It will NOT emit a warning if an atom
+%%      named as a callback is being used, no matter for what that atom is used.</p>
```

Comment:
Maybe add something like _This limitation is due to the fact that there are many ways to call a function in Erlang (particularly when dynamic calls are involved)._ to explain that it's not the _desired_ behavior but it is what what we were _able_ to implement.